packages/core/docs/classes/index.libp2p-1.md

```diff
@@ -0,0 +1,848 @@
+[@hoprnet/hopr-core](../README.md) / [Exports](../modules.md) / [index](../modules/index.md) / LibP2P
+
+# Class: LibP2P
```

Comment:
We probably don't need to document that, a link to e.g. js-libp2p might be more beneficial

Reply:
if its part of our codebase it should be included, we can add a comment in the source code for more info

Reply:
I agree. If anything the module should receive a different name to deliver its use as an improved libp2p API.